Server/EVESharp.Node/Services/Insurance/inventoryInsurancesSvc.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+
+using EVESharp.Database;
+using EVESharp.Database.Extensions;
+using EVESharp.Database.Market;
+using EVESharp.Database.Old;
+
+using EVESharp.EVE.Data.Inventory;
+using EVESharp.EVE.Data.Inventory.Items.Types;
+using EVESharp.EVE.Exceptions;
+using EVESharp.EVE.Exceptions.insuranceSvc;
+using EVESharp.EVE.Market;
+using EVESharp.EVE.Network;
+using EVESharp.EVE.Network.Services;
+using EVESharp.EVE.Network.Services.Validators;
+using EVESharp.EVE.Sessions;
+
+using EVESharp.Types;
+using EVESharp.Types.Collections;
+
+
+namespace EVESharp.Node.Services.Insurance
+{
+    [MustBeCharacter]
+    public class inventoryInsurancesSvc : ClientBoundService
```

Comment:
Separating services in two parts (bound and base) should only be relegated to big services and chains of bound services (like Inventories), otherwise it gets confusing, beacuase inventoryInsurancesSvc is not a thing in the client.